### DISCUSSION TITLE
encryptString on create / update + decryptString on setupShowOperation

### DISCUSSION THREAD

**Discussion by Astriel** [Laravel-Backpack/community-forum#755] [General]
Hello everyone. 

On my database, I've got some values which are seeded via code and encrypted with the "encryptString" method of Laravel.

I was wondering, if there was a way to encryptString (but only specific fields) on create or update (setupCreateOperation / setupUpdateOperation) without having to rewrite the whole method. 

Same question for the setupShowOperation : is there a quick way to show decrypted values which are already encrypted ?

Thanks a lot.

**Comment by pxpm** [comment]
Hey @Astriel I think that would be a good use case for a cast in the model ? https://laravel.com/docs/10.x/eloquent-mutators#encrypted-casting

Let me know if that helps 🙏

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
